vidarr-server/src/main/resources/ca/on/oicr/gsi/vidarr/server/api-docs/vidarr.json

```diff
@@ -1300,18 +1335,24 @@
     "/api/workflow/{name}/{version}": {
       "post": {
         "description": "Create a workflow version. The workflow must previously have been created.",
+        "parameters": [{
+          "name": "name",
+          "in": "path",
+          "description": "Name of workflow",
+          "required": true,
+          "type": "string"
+        }, {
+          "name": "version",
+          "in": "path",
+          "description":  "Version of workflow to create",
+          "required": true,
+          "type": "string"
+        }],
         "requestBody": {
           "content": {
             "application/json": {
               "schema": {
                 "properties": {
```

Comment:
This also can take `"accessoryFiles": {"type":"object", "additionalProperties":{"type":"string"}, "description":"An supplemental files required by the workflow (e.g., other workflows that are imported or included by the main workflow)"}`

Reply:
Added!